bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -1809,12 +1809,17 @@ func (gcp *Gcp) ConfigPodDefault() error {
 		return kfapis.NewKfErrorWithMessage(err, "set K8s clientset error")
 	}
 
+	defaultNamespace := kftypesv3.EmailToDefaultName(gcp.kfDef.Spec.Email)
+	_, err = k8sClient.CoreV1().Namespaces().Get(defaultNamespace, metav1.GetOptions{})
+	if err != nil {
+		log.Warnf("Default namespace %v creation skipped", defaultNamespace)
+		return nil
+	}
```

Comment:
why this?  the default namespace is created with the default profile in kustomize.go, right?

Reply:
Currently default profile is one entry application entry from manifest. If user delete this entry default namespace will not be created.